README.md

```diff
@@ -27,6 +29,14 @@ The focus is not on the complexity of the model, but rather on how to use the bi
 
 [Back to the table of contents](#table-of-contents)
 
+## Prerequisites
+
+The DAML code in this repository makes use of the SDK 0.12.20 and the `daml` SDK assistant.
+
+You are therefore strongly advised to have at least the SDK 0.12.20 installed.
```

Comment:
I think this should be more than a strong advice, as it seems that SDK 0.12.20 is more like a strict requirement?

Reply:
Address by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/0ebe6ca939825c0e9622d2ee6311d97dc371a9ad